### PR TITLE
Spl confirmation

### PR DESCRIPTION
**Pull request by enriquesouza**
https://omniatech.io/pages/solana-transaction-confirmation/

https://solscan.io/tx/2uLKhECS2yYxzF5oLtpYsGZeHqdQNFwNKr7xRnffPKpUEd8NXeeULru5CkJK8UW7z6ZEAG65uhnYXZpPRUhSx9P2

https://solscan.io/tx/4tb2p4HWDp4FbkhiDkbWvfYgss5UJzxDrc9x4JCNCoUL5ZezyateBBafjfopKTtk6e3TT6qCSF68pxoeFFHuBYQm

### CODE REVIEW

File: frontend/src/services/Rpc/solana/RpcServiceSolana.ts [hunk 1]
```diff
@@ -64,12 +244,24 @@ export class RpcServiceSolana implements IRpcService, ITokenService {
...
       method: 'getLatestBlockhash',
-      params: [{ commitment: 'finalized' }],
+      params: [{ commitment: 'finalized' }], //Official Solana recommendation: Use confirmed commitment when fetching recent blockhash. This will provide a better (extended with aprox. 13s) time window as compared to finalized commitment, thus reducing the risk of transaction expiration (see below).
+    };
+
+    const response = await this.postRequest(rpcURL, requestBody);
+    return response.result?.value?.blockhash as string;
+  }
+
+  static async fetchRecentBlockhash(): Promise<string> {
```
Comment (johnnyluo) [inline frontend/src/services/Rpc/solana/RpcServiceSolana.ts:259]: why we have to function with exactly the same name here?

File: frontend/src/services/Blockchain/solana/BlockchainServiceSolana.ts [hunk 2]
```diff
@@ -168,10 +174,13 @@ export class BlockchainServiceSolana
...
         }),
-        recentBlockhash: recentBlockHash,
+        recentBlockhash: newRecentBlockHash,
         sender: keysignPayload.coin.address,
         priorityFeePrice: TW.Solana.Proto.PriorityFeePrice.create({
-          price: Long.fromString(priorityFee),
+          price: Long.fromString(priorityFeePrice.toString()),
+        }),
+        priorityFeeLimit: TW.Solana.Proto.PriorityFeeLimit.create({
+          limit: Number(20000),
```
Comment (johnnyluo) [inline frontend/src/services/Blockchain/solana/BlockchainServiceSolana.ts:183]: we don't set prioirtyfeelimit in IOS as far as I understand , won't this cause the problem with signing?

File: frontend/src/services/Rpc/solana/RpcServiceSolana.ts (hunk 1 above)
Comment (enriquesouza) [reply frontend/src/services/Rpc/solana/RpcServiceSolana.ts:263]: I was testing it. To try to call it on while signing. So I had to access it externally.

File: frontend/src/services/Blockchain/solana/BlockchainServiceSolana.ts (hunk 2 above)
Comment (enriquesouza) [reply frontend/src/services/Blockchain/solana/BlockchainServiceSolana.ts]: I've added in the next PR to IOS as well )

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**